1-tictactoe/tictactoe.rb

```diff
@@ -0,0 +1,90 @@
+class Tictactoe
+
+  def play
+    # Game instructions
+    puts "Welcome Players in the TicTacToe game!
+    To hit desired position just type cell address as shown bellow."
+    positions = %w(A1 A2 A3 B1 B2 B3 C1 C2 C3)
+    print_board(positions)
+
+    # Start game
+    puts "Lets play!"
+    current_player = "X"
+    hits = new_board
+    until check_winer(hits, current_player) do
+      current_player == "X" ? current_player = "O" : current_player = "X"
+      print "Player #{current_player} turn. Type cell address: "
+      address = gets.chomp
+      valid_shot = check_position(address, hits)
+      until valid_shot do
+        print "You gave wrong address, try one more time: "
+        address = gets.chomp
+        valid_shot = check_position(address, hits)
+      end
+      update_hits(valid_shot, hits, current_player)
+      print_board(hits)
+    end
+    if board_full?(hits)
+      puts "That's a tie!"
+    else
+      puts "Game won by: Player #{current_player}!"
+    end
+  end
+
+  private
+
+  def new_board
+    Array.new(9, " ")
+  end
+
+  def print_board(hits)
+    puts "    -------------
```

Comment:
Check out heredoc for outputting multiline strings in Ruby https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc